src/lib.rs

```diff
@@ -1036,8 +1069,7 @@ impl Context {
         };
         let ret = Error::from_tss_rc(ret);
         if ret.is_success() {
-            let policy_digest = unsafe { MBox::<TPM2B_DIGEST>::from_raw(policy_digest_ptr) };
-            Ok(policy_digest.buffer[..policy_digest.size as usize].to_vec())
+            Ok(unsafe { Digest::try_from(*policy_digest_ptr)? })
```

Comment:
I am a little bit worried about removing the `MBox` here. The reason of it, I think, was to free the memory that was allocated at `policy_digest_ptr` by the callee. I am afraid we would leak memory if removed?
Would it be possible to keep that line (1039) and dereference `policy_digest` in the `Digest::try_from`? That way, when dropped, the `MBox` would still free the memory.

Reply:
Thanks for the feedback, fixed that!